pkg/asset/agent/image/agentimage.go

```diff
@@ -74,7 +75,9 @@ func (a *AgentImage) fetchAgentTuiFiles(releaseImage string, pullSecret string,
 		Config{MaxTries: OcDefaultTries, RetryDelay: OcDefaultRetryDelay},
 		releaseImage, pullSecret, mirrorConfig)
 
-	agentTuiFilenames := []string{"/usr/bin/agent-tui", "/usr/lib64/libnmstate.so.*"}
+	libnmstateFull := "libnmstate.so.2.2.9"
+	libnmstate := "libnmstate.so.2"
+	agentTuiFilenames := []string{"/usr/bin/agent-tui", filepath.Join("/usr/lib64/", libnmstateFull)}
```

Comment:
I think it can be done in a more generic way and still keep the wildcard at line 77. We should still support libnmstate.1 until the CI builds are updated and also libnmstate.so.2.2.10 is available soon.

For example, perhaps use
libnmstateFull := "libnmstate.so.2.*"
libnmstateOveride := "libnmstate.so.2"

And keep agentTuiFilenames the same at line 77 with the wildcard.
Then at line 94 do a Glob match on libnmstateFull and only override in that case.

Reply:
I'm not convinced that the effort to support libnmstate.1 is worth the game, considering that it's present only in the CI payload for mistake (given that it's  hasn't been yet updated).
The glob on the minor version seems a good idea I'll update it.